src/styles/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  margin-right: 4px;
```

Comment:
1 - due to BEM you can't apply margins to BEM blocks
2 - I think you wanted to apply this property to other class 🙂

![Screenshot 2022-08-23 at 11 12 28](https://user-images.githubusercontent.com/84275496/186107727-47b635d4-aca5-4ab7-b123-34bb4ddcff0f.png)

Reply:
Yes, you are right)
Done